content/en/getting_started/agent/_index.md

```diff
@@ -12,90 +12,219 @@ further_reading:
+| **datadog.agent.started**        | A count sent with a value of `1` when the Agent starts (available in v6.12+).                                        |
+
+See the [Agent Metrics][3] integration for a full list of Agent metrics.
+
+#### Checks
+
+Depending on your platform, the Agent has several core checks enabled by default that collect metrics.
+
+| Check       | Metrics       | Platforms          |
+| ----------- | ------------- | ------------------ |
+| CPU         | [System][4]  | All                |
+| Disk        | [Disk][5]    | All                |
+| IO          | [System][4]  | All                |
+| Memory      | [System][4]  | All                |
+| Network     | [Network][6] | All                |
+| NTP         | [NTP][7]     | All                |
+| Uptime      | [System][4]  | All                |
+| File Handle | [System][4]  | All except Mac     |
+| Load        | [System][4]  | All except Windows |
+| Docker      | [Docker][8]  | Docker             |
+| Winproc     | [System][4]  | Windows            |
+
+To collect metrics from other technologies, see the [Integrations][9] page.
+
+## Containerized Environments
+
+There are a few differences in the way the Agent runs in a containerized environment.
+
+1. The Agent configuration options are passed in with [Environment Variables][10], for example:
+    - `DD_API_KEY` for the Datadog API key
+    - `DD_SITE` for the Datadog site
+
+2. Integrations are automatically identified through Datadog's Autodiscovery feature. See [Basic Agent Autodiscovery][11] to learn more.
+
+See the [Docker Agent][12] or [Kubernetes][13] for a walkthrough on running the Agent in a containerized environment.
+
+## Why Should I Install the Agent?
+
+The Agent needs to be installed to pull data from any one of the many Agent based Integraions. The Agent is not necessarily required to forward data to the Datadog Platform, for example, you can send Logs and Metrics through the Datadog API. However, the Agent is the recommended method to forward your data to the Datadog Platform. 
+
+The Agent samples host data  every 15 seconds to provide a more accurate understanding of what is happening across your environments. Additionally, it comes with over 50 default metrics to provide greater insight on system level data.
 
 ## Setup
 
-If you haven't already, create a [Datadog account][7].
+### Prerequisites
 
-### Installation
+1. Create a [Datadog account][14].
 
-The Agent can be installed on many different platforms either directly on the host or as a [containerized version][8]. Most systems have a one-line install option.
+2. You will need access to your [Datadog API key][15]. 
 
-{{< partial name="platforms/platforms.html" desc="Choose your platform to see installation instructions:" links="gs" >}}
+3. Have the Datadog UI open.
 
-### Configuration
+**Note** This walkthrough will be using the Ubuntu platform. See the [Basic Agent Usage][16] page for a full list of supported platforms. 
```

Comment:
I would say "Ubuntu operating system" instead of "platform". It may also make sense to specify the version of the OS, like "Ubuntu 20.04"

Reply:
Agreed about using "operating system", however we don't want to give the impression that this process only works on one very specific version. 

Also, colon after **Note**: